libdrgn/program.c

```diff
@@ -50,6 +50,27 @@ drgn_program_platform(struct drgn_program *prog)
 	return prog->has_platform ? &prog->platform : NULL;
 }
 
+void drgn_program_init_language(struct drgn_program *prog)
+{
+	struct drgn_error *err;
+	struct drgn_qualified_type main_type;
+	struct drgn_object res;
+
+	drgn_object_init(&res, prog);
+
+	// Use the CU from the "main" object as the lang for the whole program.
+	err = drgn_program_find_object(prog, "main", NULL,
```

Comment:
@osandov Actually, one more thing I forgot to ask. I *think* as it is right now, this line will populate the dwarf info cache which (I think) is currently meant to be loaded lazily. If that's the case, I could try to load this lazily as well. What do you think?

Reply:
At this point, the debugging information has already been loaded and the DWARF index populated, so it's fair game to look things up.